index.html

```diff
@@ -75,5 +75,10 @@ <h4 class="box-title">Django </h4>
       </div>
     </div>
   </div>
+  <header class="footer">
+    <p class="footer-title">WE ARE GAME CHANGERS</p>
```

Comment:
the uppercase is something aesthetic so it should be done in css using `text-transform: uppercase`